.travis.yml

```diff
@@ -4,12 +4,22 @@ env:
   global:
     - SCAN_SKIP_BUILD=1
   matrix:
-    # Define the details for each test suite that should be run
-    - SCAN_WORKSPACE="SampleProject/SampleProject.xcworkspace"                           SCAN_SCHEME="VOKCoreDataManager"         SCAN_SDK="iphonesimulator"
+    # Test on iOS 10.2
+    - SCAN_WORKSPACE="SampleProject/SampleProject.xcworkspace"                           SCAN_SCHEME="VOKCoreDataManager"         SCAN_DEVICE="iPhone 5 (10.2)"
```

Comment:
Oh right, I forgot that [32-bit support isn't really a thing](https://arstechnica.com/apple/2017/01/future-ios-release-will-soon-end-support-for-unmaintained-32-bit-apps/?utm_campaign=iOS%2BDev%2BWeekly&utm_medium=email&utm_source=iOS_Dev_Weekly_Issue_286) anymore anyway.